packages/contracts-bedrock/scripts/libraries/ForgeArtifacts.sol

```diff
@@ -108,19 +104,16 @@ library ForgeArtifacts {
         // functional for now. Deployment script should make this easier to determine.
         string memory res = Process.bash(
             string.concat(
-                "jq",
-                " -r '.rawMetadata' ",
+                "jq -r '.rawMetadata' ",
                 _getForgeArtifactPath(_name),
-                " | ",
-                "jq",
-                " -r '.output.devdoc' | jq -r 'has(\"custom:predeploy\")'"
+                " | jq -r '.output.devdoc' | jq -r 'has(\"custom:predeploy\")'"
             )
         );
         out_ = stdJson.readBool(res, "");
     }
 
     function _getForgeArtifactDirectory(string memory _name) internal returns (string memory dir_) {
-        string memory res = Process.bash(string.concat("forge", " config --json | ", "jq", " -r .out"));
+        string memory res = Process.bash(string.concat("forge config --json | jq -r .out"));
```

Comment:
No need for `concat` here anymore. Worth checking for other places where this might be true too.

Reply:
ah i missed this

Reply:
fixed